app/uk/gov/hmrc/cipphonenumberverification/services/NotificationService.scala

```diff
@@ -55,20 +57,25 @@ class NotificationService @Inject()(govNotifyUtils: GovNotifyUtils, auditService
         case "temporary-failure" => (107, "Message was unable to be delivered by the network provider")
         case "technical-failure" => (108, "There is a problem with the notification vendor")
       }
-      auditService.sendExplicitAuditEvent(AuditType.PHONE_NUMBER_VERIFICATION_DELIVERY_RESULT_REQUEST.toString,
+      auditService.sendExplicitAuditEvent(PhoneNumberVerificationDeliveryResultRequest,
         VerificationDeliveryResultRequestAuditEvent(phoneNumber, passcode, notificationId, deliveryStatus))
       Ok(Json.toJson(NotificationStatus(code, message)))
     }
 
     def failure(err: UpstreamErrorResponse) = {
       err.statusCode match {
         case NOT_FOUND =>
-          logger.warn("Notification ID not found")
-          auditService.sendExplicitAuditEvent(AuditType.PHONE_NUMBER_VERIFICATION_DELIVERY_RESULT_REQUEST.toString,
+          logger.warn("Notification Id not found")
+          auditService.sendExplicitAuditEvent(PhoneNumberVerificationDeliveryResultRequest,
             VerificationDeliveryResultRequestAuditEvent(NO_DATA_FOUND, NO_DATA_FOUND, notificationId, NO_DATA_FOUND))
           NotFound(Json.toJson(ErrorResponse(Codes.NOT_FOUND, NO_DATA_FOUND)))
```

Comment:
I know that this was not part of your ticket but I am wondering if this should be changed back to:

      (response.json \ "code").as[String] shouldBe "NOTIFICATION_NOT_FOUND"
      (response.json \ "message").as[String] shouldBe "Notification ID not found"

Reply:
lets discuss this tomorrow, not sure what you mean?

Reply:
NO_DATA_FOUND is for the audit event.  we agreed to have the same in the response message field if that is what you mean?
NO_DATA_FOUND = "No_data_found"